give.php

```diff
@@ -755,6 +749,24 @@ private function is_request( $type ) {
 			}
 		}
 
+		/**
+		 * Handle property get request
+		 *
+		 * @param string $propertyName
+		 *
+		 * @since 2.7.0
+		 * @return mixed
+		 */
+		function __get( $propertyName ) {
+			if ( 'themes' === $propertyName ) {
+				if ( ! $this->themes instanceof Themes ) {
+					$this->themes = new Themes();
+				}
+
+				return $this->themes;
+			}
```

Comment:
So is this temporary until you get the other PR merged? Ideally you wouldn't have to list out the properties one by one in here, but have an array property you're able to reference (i.e. `$this->aliases`). Also, use something like a `$singletonsCache` array property. Avoid dynamic properties as they're a good way to run into unintentional collisions. So if this is temporary something like this:
```php
			if ( 'themes' === $propertyName ) {
				if ( ! isset($this->singletonCache[Themes::class] ) {
					$this->singletonCache[Themes::class] = new Themes();
				}

				return $this->singletonsCache[Themes::class]
			}
```

Reply:
@JasonTheAdams This was the only property that why I setup like it but I will use an array to store singleton and yes it is a temporary solution.

Reply:
fix: https://github.com/impress-org/givewp/compare/c96a437a6320...b6184d567284